test/extended/testdata/router/router-config-manager.yaml

```diff
@@ -1,275 +0,0 @@
-apiVersion: template.openshift.io/v1
-kind: Template
-parameters:
-- name: IMAGE
-  value: openshift/origin-haproxy-router:latest
-objects:
-- apiVersion: v1
-  kind: Pod
-  metadata:
-    name: router-haproxy-cfgmgr
-    labels:
-      test: router-haproxy-cfgmgr
-  spec:
-    terminationGracePeriodSeconds: 1
-    containers:
-    - name: router
-      image: ${IMAGE}
-      imagePullPolicy: IfNotPresent
-      env:
-      - name: POD_NAMESPACE
-        valueFrom:
-          fieldRef:
-            fieldPath: metadata.namespace
-      args: ["--namespace=$(POD_NAMESPACE)", "-v=4", "--haproxy-config-manager=true", "--blueprint-route-labels=select=hapcm-blueprint", "--labels=select=haproxy-cfgmgr", "--stats-password=password", "--stats-port=1936", "--stats-user=admin"]
-      hostNetwork: false
-      ports:
-      - containerPort: 80
-      - containerPort: 443
-      - containerPort: 1936
-        name: stats
-        protocol: TCP
-    serviceAccountName: default
-
-# ensure the router can access routes and endpoints
-- apiVersion: v1
-  kind: RoleBinding
-  metadata:
-    name: system-router
-  subjects:
-  - kind: ServiceAccount
-    name: default
-  roleRef:
-    name: system:router
-
-# blueprints for edge, reencrypt and passthrough routes with annotation(s)
-- apiVersion: route.openshift.io/v1
-  kind: Route
-  metadata:
-    name: edge-blueprint
-    labels:
-      test: router
-      select: hapcm-blueprint
-    annotations:
-      router.openshift.io/cookie_name: empire
-  spec:
-    tls:
-      termination: edge
-    host: edge.blueprint.hapcm.test
-    to:
-      name: insecure-service
-      kind: Service
-    ports:
```

Comment:
Wow, I just realized that this has been wrong all these years—it should be `port`, not `ports`.  Fortunately `spec.port` is only needed if the target service has multiple ports (when `spec.port` is omitted, the router distributes traffic over all the service's ports).  Anyway, I'm going to use this as yet another example of why replacing yaml assets with type-checked Go definitions is a good idea.